packages/connect-examples/exampleV2/app/components/ui/ActionButton.tsx

```
@@ -0,0 +1,46 @@
+      "bg-white border border-gray-300 text-gray-700 hover:bg-gray-50 hover:shadow-xl",
+  };
+
+  return (
+    <Button
+      className={`py-4 text-lg font-semibold rounded-full shadow-lg px-6 transition-all duration-200 hover:opacity-90 active:scale-95 ${variantClasses[variant]} ${className}`}
+      onClick={onClick}
+      disabled={disabled || isLoading}
+    >
+      {isLoading ? (
+        <div className="flex items-center space-x-2">
+          <div className="animate-spin rounded-full h-4 w-4 border-b-2 border-current"></div>
```

Biome (1.9.4)

[error] JSX elements without children should be marked as self-closing. In JSX, it is valid for any element to be self-closing.

Unsafe fix: Use a SelfClosingElement instead


(lint/style/useSelfClosingElements)